.env.sample

```diff
@@ -1,43 +0,0 @@
-NODE_ENV=development
```

Comment:
@ydahal1 - Was this file removed intentionally?

Reply:
I don't remember doing that. It is unintentional